spec/system/custom_sidebar_sections_spec.rb

```diff
@@ -168,23 +168,28 @@
     sidebar.click_add_section_button
     sidebar.click_add_link_button
 
-    is_focused =
-      page.evaluate_script("document.activeElement.classList.contains('multi-select-header')")
```

Comment:
This does not rely on any capybara helpers that automatically retries for a set amount of time.